app/src/main/res/layout/browser_gesture_wrapper.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<org.mozilla.fenix.browser.SwipeGestureLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/gestureLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <org.mozilla.fenix.browser.TabPreview
```

Comment:
Does the ordering of these matter? Before the TabPreview was underneath the coordinatorlayout and now I think it will be above it?

Reply:
I haven't noticed any difference in behavior.